userguide/sharing.rst

```diff
@@ -1840,44 +1839,34 @@ initiator.
-
-
-In the example shown in
-:numref:`Figure %s <iscsi_initiator_conf_sample_fig>`,
-two groups are created. Group 1 allows connections from any
-initiator on any network. Group 2 allows connections from any
-initiator on the *10.10.1.0/24* network. Click |ui-options| on an
-initiator entry to display its :guilabel:`Edit` and :guilabel:`Delete`
-buttons.
-
-.. note:: Attempting to delete an initiator causes a warning that
-   indicates if any targets or target/extent mappings depend upon the
-   initiator. Confirming the delete causes these to be deleted also.
-
-
-.. _iscsi_initiator_conf_sample_fig:
-
-.. figure:: images/sharing-block-iscsi-initiators-example.png
-
-   Sample iSCSI Initiator Configuration
+   +----------------------+-----------+--------------------------------------------------------------------------------------+
+   | Setting              | Value     | Description                                                                          |
+   |                      |           |                                                                                      |
+   +======================+===========+======================================================================================+
+   | Allow All Initiators | checkbox  | Accept all detected initiators. When set, all other initiator fields are disabled.   |
+   +----------------------+-----------+--------------------------------------------------------------------------------------+
+   | Connected Initiators | string    | Initiators currently connected to the system. Shown in IQN format with an IP         |
+   |                      |           | address. Set initiators and click an |arrow-right| to add the initiators to either   |
+   |                      |           | the :guilabel:`Allowed Initiators` or :guilabel:`Authorized Networks` lists.         |
+   |                      |           | Clicking :guilabel:`REFRESH` updates the :guilabel:`Connected Initiators` list.      |
+   +----------------------+-----------+--------------------------------------------------------------------------------------+
+   | Allowed Initiators   | string    | Initiators allowed access to this system. Enter an                                   |
+   | (IQN)                |           | `iSCSI Qualified Name (IQN) <https://tools.ietf.org/html/rfc3720#section-3.2.6>`__   |
+   |                      |           | and click :guilabel:`+` to add it to the list. Example:                              |
+   |                      |           | :samp:`{iqn.1994-09.org.freebsd:freenas.local}`                                      |
+   +----------------------+-----------+--------------------------------------------------------------------------------------+
+   | Authorized Networks  | string    | Network addresses which are allowed to use this initiator. Each address can include  |
```

Comment:
This can be shorter like the previous one:
```suggestion
   | Authorized Networks  | string    | Network addresses allowed to use this initiator. Each address can include         |
```